src/test/resources/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark/SVBreakpointsTest.kmers

```diff
@@ -1,356 +1,345 @@
-AATTATATTCCAGAGTACAAAAGAAATTTGCAGATAGAATCACAGAACCAC
```

Comment:
Why did these results change? I thought this PR was pretty much just a performance improvement.